Source/Core/DolphinQt2/GameList/TableDelegate.cpp

```diff
@@ -52,24 +58,44 @@ void TableDelegate::paint(QPainter* painter, const QStyleOptionViewItem& option,
 
 QSize TableDelegate::sizeHint(const QStyleOptionViewItem& option, const QModelIndex& index) const
 {
+  const auto scale = DPI::GetScale();
   switch (index.column())
   {
   case GameListModel::COL_PLATFORM:
-    return Resources::GetPlatform(0).size();
+    return Resources::GetPlatform(0).size() / Resources::GetIconScale() * scale;
```

Comment:
I couldn't address this up to this point, as your PR was not merged at the time of the commit. Will take a look at it now however.